zombies.js

```diff
@@ -79,6 +90,99 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+function Player(name, health, strength, speed) {
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+  this.speed = speed;
+  this._pack = [];
+  this._maxHealth = this.health;
+  this.isAlive = true;
+  this.equipped = false;
+
+  this.getPack = function() {
+    return this._pack;
+  };
+
+  this.getMaxHealth = function() {
+    return this._maxHealth;
+  };
+
+  this.takeItem = function(item) {
```

Comment:
All the function below should be on Player's prototype and not in the constructor function. The way it currently is, these methods won't get inherited if someone extends player